File: Views/src/components/services-provision/budget/dto/service-budget-dto.ts
import { CustomerDto } from "src/components/main/customer/dto/customer-dto";
import { SolutionPriceDto } from "../../dtos/solution-price-dto";

export class ServiceBudgetDto {
  id: number;
  customerId: number;
  customer: CustomerDto;
  budgetStartedIn: Date;
  visually: string;
  remoteAccessData: string;
  customerProblems: string;
  benchStartedIn: Date;
  status: string;
  authorized:boolean;
  solutionsPrices: SolutionPriceDto[]
}

File: Application/Dto/ServiceBudgetBench/ServiceBudgetDto.cs
using System;
using System.Collections.Generic;

namespace Application.Dto.ServiceBudgetBench

{
    public class ServiceBudgetDto
    {
        public int Id { get; set; }
        public int CustomerId { get; set; }
        public CustomerDto Customer { get; set; }
        public DateTime BudgetStartedIn { get; set; }
        public string Visually { get; set; }
        public string RemoteAccessData { get; set; }
        public string CustomerProblems { get; set; }
        public DateTime BenchStartedIn { get; set; }
        public string Status { get; set; }
        public bool Authorized { get; set; }
        public List<SolutionPriceDto> SolutionsPrices { get; set; }
    }
}
